pkg/kinflate/util.go

```diff
@@ -17,63 +17,17 @@ limitations under the License.
 package manifest
 
 import (
-	"errors"
 	"io/ioutil"
-	"path"
 
 	"gopkg.in/yaml.v2"
 
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 	manifest "k8s.io/kubectl/pkg/apis/manifest/v1alpha1"
-	"k8s.io/kubernetes/pkg/kubectl/resource"
 )
 
 const kubeManifestFileName = "Kube-manifest.yaml"
 
-// loadBaseAndOverlayPkg returns:
-// - List of FilenameOptions, each FilenameOptions contains all the files and whether recursive for each base defined in overlay kube-manifest.yaml.
-// - Fileoptions for overlay.
-// - Package object for overlay.
-// - A potential error.
-func loadBaseAndOverlayPkg(f string) ([]resource.FilenameOptions, resource.FilenameOptions, *manifest.Manifest, error) {
```

Comment:
Don't remove it.

Comment it for now :)

Reply:
Yeah, I'll comment ;-)

Reply:
Done